CONTRIBUTING.md

```diff
@@ -69,6 +69,9 @@ Minimizing developer follow-up action is [helpful](#positive-helpful-feedback)!
 
 If no Google Cloud resources are needed, use [GitHub Actions](https://docs.github.com/en/actions) to drive automation. Otherwise use Cloud Build ([decision](docs/decisions/2021-05-static-analysis.md)).
 
+Infrastructure (including test fixtures) should be managed by Terraform wherever practical. In cases where resources **cannot** be managed by Terraform, this project recommends manual management instead
+of custom scripts/automation. ([decision](docs/decisions/2022-07-pet-test-fixtures.md))
```

Comment:
This isn't quite true: we actively use scripts/automation in many places for one-time setup. Where we've avoided automation is project creation, and that's primarily a matter of security policy.

Reply:
Reworded; PTAL?